src/rshim_fuse.c

```diff
@@ -680,7 +680,17 @@ static int rshim_fuse_misc_read(struct cuse_dev *cdev, int fflags,
   p += n;
   len -= n;
 
-  /* SW reset flag is always 0. */
+  n = snprintf(p, len, "%-16s%d (0:no access, 1:has access)\n", "RSHIM ACCESS",
```

Comment:
The misc file os key/value style where the first string is keyword. Can we change it to be "RSHIM_ACCESS" instead of the space separated "RSHIM ACCESS"?

Reply:
Will change.

Reply:
fixed in 269e388